VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingWrapPanel.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.0;net45</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;net45</TargetFrameworks>
```

Comment:
What is the advantage of targeting netcoreapp3.0 and netcoreapp3.1?

Reply:
Just for compatibility. I prefer to target to netcoreapp3.1.